src/main/resources/config/liquibase/changelog/20240418092928_add_new_field.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet id="00000000000002" author="beyzaaltuntas">
+        <addColumn tableName="quiz_question">
+            <column name="content" type="varchar(5000)">
+                <constraints nullable="false"/>
```

Comment:
This will not work. This is an already existing table, which already contains columns in production. So setting this to nullable is not possible, as this constraint would be violated for all existing data.
Either you will have to calculate defaults or make it nullable.